### PR TITLE
Applied fixes from FlintCI

### PR DESCRIPTION
**Pull request by soullivaneuh**

### CODE REVIEW

File: tests/Controller/CRUDControllerTest.php [hunk 1]
```diff
@@ -149,9 +148,6 @@ class CRUDControllerTest extends TestCase
      */
     private $translator;
 
-    /**
-     * @var LoggerInterface&MockObject
-     */
```
Comment (greg0ire) [inline tests/Controller/CRUDControllerTest.php:154]: Not sure about that one…

Comment (core23) [reply tests/Controller/CRUDControllerTest.php:154]: Looks like a bug in the php-cs-fixer version

Comment (greg0ire) [reply tests/Controller/CRUDControllerTest.php:154]: Here are the rules that are applied when I apply everything but this: `no_superfluous_phpdoc_tags, no_empty_phpdoc, no_whitespace_in_blank_line, no_extra_blank_lines`

Comment (greg0ire) [reply tests/Controller/CRUDControllerTest.php:154]: See https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4913

Comment (greg0ire) [reply tests/Controller/CRUDControllerTest.php:154]: Looks like there is currently no way to pin a flintCI container to a specific minor or patch version of php-cs-fixer: https://hub.docker.com/r/flintci/php-cs-fixer/tags

**Comment by greg0ire** [comment]
Closed by #6047

**Pull request closed without merging**